modules/uninstalling-olm-using-ansible.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * install_config/uninstall-olm-using-ansible.adoc
+
+[id="uninstalling-olm-using-ansible_{context}"]
+= Uninstalling Operator Lifecycle Manager using Ansible
+
+After installing your cluster, you can uninstall the Technology Preview Operator Framework by using he {product-title} `openshift-ansible` installer.
+
+.Prerequisites
```

Comment:
I also don't think we use the Prereq and Procedure headings in 3.11, but I'm not sure if that's a big deal to include or not